src/serac/numerics/functional/tests/tuple_arithmetic_unit_tests.cpp

```diff
@@ -144,11 +144,9 @@ TEST(ChainRuleTests, scalar_output_with_matrix_input)
   EXPECT_NEAR(((f(x + epsilon * dx) - f(x)) / epsilon) - chain_rule(dfdx, dx), 0.0, 1.e-6);
 }
 
-TEST(ChainRuleTests, tuple_output_with_tuple_input)
+TEST(ChainRuleTests, tensor_output_with_tuple_input)
```

Comment:
It's very helpful to have the filename as the first parameter so when one fails you know which file to go into to fix it.